pkg/scd/operational_intents_handler.go

```diff
@@ -508,6 +508,56 @@ func validateUpsertRequestAgainstPreviousOIR(
 	return nil
 }
 
+func incrementIndicesAndGetRelevantSubscriptions(
```

Comment:
nit: I think the primary thing this function is doing is getting relevant subscriptions, with a sub-behavior of incrementing notification indices

```suggestion
func getRelevantSubscriptionsAndIncrementIndices(
```